### ISSUE TITLE
nuxt ssr support?

### ISSUE THREAD

**Issue by reslear** [swayable/vue-snarkdown#1]
hello 
does support this wrapper in nuxt ssr ?

**Comment by jdbdnz** [comment]
That's a good question! I don't see any reason why it shouldn't. Did you try it out?

**Comment by FreekVR** [comment]
Hi! I wanted to use this package with Nuxt SSR too, but there's a minor issue, seems to be that the UMD build (which is used by node / ssr) is not linked correctly:

package.json
```
  "main": "dist/vue-snarkdown.js",
  "module": "dist/vue-snarkdown.esm.js",
  "unpkg": "dist/vue-snarkdown.min.js",
  "browser": {
    "./sfc": "src/vue-snarkdown.js"
  },
```

In the `/dist/` there is no "dist/vue-snarkdown.js",, I believe this should be ""dist/vue-snarkdown.umd.js", and the browser/sfc version should also use a different file (the *.min.js I guess).

My current workaround is to import UMD file always, directly, like so:

`import Snarkdown from '@swayable/vue-snarkdown/dist/vue-snarkdown.umd.js';`

This appears to work fine in both client- and server context.

**Comment by jdbdnz** [comment]
Thanks @FreekVR for the investigation and report. Your suggestions are included in v2.0.0.

If you can, try out a regular ol' `import Snarkdown from '@swayable/vue-snarkdown'` and let me know if it works as expected.
